mon-pix/app/styles/components/_challenge-statement.scss

```diff
@@ -157,8 +156,8 @@
   }
 
   input[type=radio]:checked:after {
-    top: -19px;
-    left: 2px;
+    top: -17px;
+    left: 0px;
```

Comment:
On peut mettre `left: 0;` , c'est plus simple 
(y'a du détail dans la première réponse: https://stackoverflow.com/questions/7923042/when-specifying-a-0-value-in-css-should-i-explicitly-mark-the-units-or-omit )

Reply:
Avec ce nouveau style, j'obtiens un décalage du point central sur mon radio button (challengeId: recTrVR10C6nIraJU):

<img width="741" alt="Screenshot 2020-02-03 at 11 16 17" src="https://user-images.githubusercontent.com/11294487/73644959-a1089180-4676-11ea-9d46-e221b1a20ffa.png">

Reply:
Modifié :)